alpine/pom.xml

```diff
@@ -0,0 +1,478 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ This file is part of Alpine.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  ~ SPDX-License-Identifier: Apache-2.0
+  ~ Copyright (c) Steve Springett. All Rights Reserved.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.dependencytrack</groupId>
+        <artifactId>dependency-track-parent</artifactId>
+        <version>5.6.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>alpine-parent</artifactId>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>alpine-common</module>
+        <module>alpine-model</module>
+        <module>alpine-infra</module>
+        <module>alpine-server</module>
+        <module>alpine-executable-war</module>
+    </modules>
+
+    <name>alpine-parent</name>
+    <description>
+        An opinionated scaffolding library that jumpstarts Java projects with an API-first design,
+        secure defaults, and minimal dependencies.
+    </description>
+    <url>https://github.com/stevespringett/Alpine</url>
+    <inceptionYear>2016</inceptionYear>
+
+    <licenses>
+        <license>
+            <name>Apache-2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Steve Springett</name>
+            <organizationUrl>http://stevespringett.com/</organizationUrl>
+            <roles>
+                <role>Architect</role>
+                <role>Developer</role>
+            </roles>
+        </developer>
+    </developers>
+
+    <properties>
+        <!-- Maven Build Properties -->
+        <timestamp>${maven.build.timestamp}</timestamp>
+        <checkstyle.skip>true</checkstyle.skip>
```

Comment:
The Alpine codebase does not conform to our checkstyle rules yet, so excluding it from codestyle checks for now.